src/devices/bus/isa/mda.cpp

```diff
@@ -965,3 +975,514 @@ MC6845_UPDATE_ROW( isa8_ec1840_0002_device::crtc_update_row )
+ IC8  74LS389?                            IC33 74LS08                               IC58 74F00?
+ IC9  74F02                               IC34 74LS245                              IC59 74LS244
+ IC10 74ALS109                            IC35 74F10?                               IC60 TMS4416-15NL 4 x 16Kbits DRAM
+ IC11 Crystal 17.040MHz                   IC36 74LS02                               IC61 TMS4416-15NL 4 x 16Kbits DRAM
+ IC12 74F64                               IC37 74LS00                               IC62 74ALS574
+ IC13 74ALS299                            IC38 74F374                               IC63 74LS138
+ IC14 PAL? 10-70ART40101                  IC39 74LS125                              IC64 74LS245
+ IC15 EPROM 8363 65 14-80 CG 50821 A64    IC40 74LS244                              IC65 74LS00
+ IC16 Crystal 19.170MHz                   IC41 74LS244                              IC66 74LS02
+ IC17 74LS10                              IC42 74LS574                              IC67 74LS51
+ IC18 74F08                               IC43 74LS32                               IC68 74LS04
+ IC19 74ALS574                            IC44 MC10124 - TTL to MECL converter      IC69 74LS153
+ IC20 74LS299                             IC45 74LS109                              IC70 74LS109
+ IC21 74LS273                             IC46 74LS00                               IC71 74LS138
+ IC22 74ALS574                            IC47 74F194                               IC72 74LS139
+ IC23 CRTC HD46505SP,HD68A45SP            IC48 74F04 
+ IC24 EPROM 2764, 10-40 VP 402 28 A19     IC49 74LS174
+ IC25 74ALS109                            IC50 TMS4416-15NL 4 x 16Kbits DRAM
+
+ General description
+ -------------------
+ The PCB has a 2 bit DIP switch S1 and a DB15 non standard video connector. There is also an unsoldered J4 connector
+ above the DB15 but no hole prepared for a connector in the plate. Above the J4 connector there is a two pin PCB connector
+ that probably receives the power for the monitor for the DB15 from the PSU.
+
+ Just below IC65 and IC66 there are two labels saying "I 85565" and "A E85571" respectively
+
+ Video cable, card DB15 <---> monitor DB25
+ ---------------------------------------------------
+  Ericsson       2  +VS             4  Ericsson
+  Monochrome     3  VS return       2  Monochrome HR
+  HR Graphics	10  +VS            17  Monitors 3111 (Amber) or 
+  Board 1070	11  VS return      15  3712/3715 (Black & White)
+		 4  VSYNC           6
+		12  VSYNC          19
+		 5  HSYNC           7
+		13  HSYNC          20
+		 6  High intensity  8
+		14  High intensity 21
+		 7  Video           9
+		15  Video          22
+                 8  GND            11
+
+  This board is normaly used with an Ericsson monitor due to the non standard connector.
+  Trivia: https://www.pinterest.se/pin/203084264425177097/
+ */
+
+#define EPC_MDA_SCREEN "epc_mda_screen"
```

Comment:
You're better off passing the device finder around in the machine configuration than using a macro like this – it's less fragile.

Reply:
Yep, but the other devices uses this approach so I'll keep it until I split out epc_mda into its own device driver file, as suggested in the preliminary PR